### PR TITLE
5002-manual check execution bugfix

### PR DESCRIPTION
**Pull request by ManishaKumari295**
Closed https://github.com/sensu/sensu-go/issues/5002

**Description**
Each manual execution of a check via the web UI or sensuctl, must execute normally as it runs when scheduled.

**Change in behavior**
Manual execution of check used to throw an error of -'no matching entity found". With the changes made in context pass, the behaviour and fixed and matching entities are found.

**Added**
Nothing added ,only modification in existing function

**Changed**
The adhoc request of check execution is now processsed with the Context from resource that that an added namespace key in it.
changes are made listenQueue() and ProcessCheck() in executor.go in schedulered directory.

**Fixed**
This fixes the entity fetch for the manual executed check and is shown in logs after execution.

**Change verification**
Added log snippet of scheduler , which comes on manual check execution .
5022_log

Snippet of testcase execution is also attached.

### CODE REVIEW

**Comment by ManishaKumari295** [comment]
Test case-  TestProcessCheck() ,test result attach
5002_TC
ed

**Comment by fguimond** [comment]
Nice clean fix, the draft status can be removed. Well done!

**Comment by fguimond** [comment]
LGTM. Just make sure you sign-off your commits (`Signed-off-by: Author Name <authoremail@example.com>`.

**Comment by fguimond** [comment]
Don't forget to add a changelog entry.

**Comment by ManishaKumari295** [comment]
> Don't forget to add a changelog entry.

Added the changelog.md.

File: backend/schedulerd/executor.go [hunk 1]
```diff
@@ -306,6 +309,10 @@ func processCheck(ctx context.Context, executor Executor, check *corev2.CheckCon
 		}
 		// publish proxy requests on matching entities
 		if matchedEntities := matchEntities(entities, check.ProxyRequests); len(matchedEntities) != 0 {
+			//display matched entities
+			matchedEntityJson, _ := json.Marshal(matchedEntities)
+			matchedEntityJsonData := string(matchedEntityJson)
+			logger.WithFields(fields).Infof("matched entities: %s", matchedEntityJsonData)
```
[the remark this answers is not in the document]
Comment (ManishaKumari295) [reply backend/schedulerd/executor.go]: > I like the idea of a log here but it should probably be at the Debug level, as we might not want to view this under normal operations.

Removed the logs from code

**Pull request merged**